### DISCUSSION TITLE
Welcome to bbAir Discussions!

### DISCUSSION THREAD

**Discussion by RealCorebb** [RealCorebb/bbAir#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by runswithcode** [comment]
Hi there! Beautiful project! I was curious if you would share what valve you used for this project? I've wanted to do a volumetric version of this project for some time, but the valves I've found are generally too large or expensive. Have you considered that as well? It would be cool to do "3D image printing" with bubbles :)
